spec/features/openid_connect/authorization_confirmation_spec.rb

```diff
@@ -82,4 +82,29 @@ def create_user_and_remember_device
       expect(current_path).to eq(new_user_session_path)
     end
   end
+
+  context 'first time registration' do
```

Comment:
Would be good to have a similar spec for the SAML version of this controller, as well as the password reset flow.